subtools/fastq_sorter.py

```diff
@@ -0,0 +1,59 @@
+def filter_gc_content(seq: str, gc_bounds: tuple = ((0, 100))) -> bool:
+    """
+    Filters the sequences if their GC-content is corresponding the circumstances.
+    Keyword arguments:
+    - seq — nucleotide sequence that should be analysed;
+    - gc_bounds — the threshold of GC-content (unnecessary, (0, 100) is default).
+    Output: a boolean value (True if the secuence corresponds the values of GC-content).
+    """
+    # Counter of C and G in the sequence:
+    gc_counter = 0
+    for nucl in seq.lower():
+        if nucl == 'g' or nucl == 'c':
+            gc_counter += 1
+    # Calculation of GC-content of the sequence:
+    gc = gc_counter / len(seq) * 100
+    # Checking the correspondence of conditions:
+    if isinstance(gc_bounds, float) or isinstance(gc_bounds, int):
+        if 0 <= gc <= float(gc_bounds):
+            return True
+    else:
+        if min(float(gc_bounds)) <= gc <= max(float(gc_bounds)):
+            return True

```

Comment:
Здорово! Только:
1) Возвращаем сразу результат булевой операции
2) Тут можно было не разбивать логику на две ветки, а свести все к одной:
```suggestion
    if isinstance(gc_bounds, float) or isinstance(gc_bounds, int):
        gc_bounds = (0, gc_bounds)
    return gc_bounds[0] <= gc <= gc_bounds[1]:
```